op-batcher/flags/flags.go

```diff
@@ -189,9 +189,16 @@ var (
 	EspressoUrlFlag = &cli.StringFlag{
 		Name:    "espresso-url",
 		Usage:   "",
-		Value:   "", // should be larger than the builder's max-l2-tx-size to prevent endlessly throttling some txs
+		Value:   "",
 		EnvVars: prefixEnvVars("ESPRESSO_URL"),
 	}
+
+	EspressoLCAddrFlag = &cli.StringFlag{
```

Comment:
Nit: Not sure about Go naming conventions, but if it's similar to Rust, maybe it's better to use `Lc` rather than `LC`.

Reply:
Another nit: Can we pick either `address` or `url` for both the Espresso URL above and the light client URL here, for naming consistency?

Reply:
> Nit: Not sure about Go naming conventions, but if it's similar to Rust, maybe it's better to use Lc rather than LC.

Go actually does the opposite, i.e. either "lc" or "LC" but never "Lc": https://go.dev/wiki/CodeReviewComments#initialisms

> Another nit: Can we pick either address or url for both the Espresso URL above and the light client URL here, for naming consistency?

Those refer to different things though, one is an URL of an HTTP endpoint and another one is address of a contract on L1

Reply:
> Go actually does the opposite, i.e. either "lc" or "LC" but never "Lc": https://go.dev/wiki/CodeReviewComments#initialisms

TIL!